src/components/GWL.vue

```diff
@@ -276,12 +276,12 @@ export default {
         
         var n_quant = quant_cat.length
         var percData = [];
-          for (i = 0; i < n_quant; i++) {
-          var key_quant = quant_cat[i];
+        for (let j = 0; j < n_quant; j++) {
```

Comment:
ah so changing this var name in case it conflicts with the other instance `i` is used 👍